Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM node:20.11.1-alpine AS base
```

Comment:
we already have docker file under docker folder

Reply:
Good catch, will remove it!

Reply:
https://github.com/oss-apps/split-pro/pull/132/commits/a1caeed1f7cdf5c4546b2ff8868fa528abf0f781